spec/01-lexical-syntax.md

```diff
@@ -498,6 +499,62 @@ of the escape sequences [here](#escape-sequences) are interpreted.
 > [implicit conversion](06-expressions.html#implicit-conversions) from `String` to
 > `StringLike`, the method is applicable to all strings.
 
+#### Interpolated string
+
+```ebnf
+interpolatedString ::= alphaid ‘"’ {printableChar \ (‘"’ | ‘\$’) | escape} ‘"’ 
+                         |  alphaid ‘"""’ {[‘"’] [‘"’] char \ (‘"’ | ‘\$’) | escape} {‘"’} ‘"""’
+escape                 ::= ‘\$\$’ 
+                         | ‘\$’ id
+                         | ‘\$’ BlockExpr
+alphaid                ::= upper idrest
+                         |  varid
+
+```
+
+Interpolated string consist of an identifier starting with a letter immediately 
+followed by a string literal. There may be no whitespace characters or comments 
+between the leading identifier and the opening quote ‘”’ of the string. 
+The string literal in a interpolated string can be standard (single quote) 
+or multi-line (triple quote).
+
+Inside a interpolated string none of the usual escape characters are interpreted 
+(except for unicode escapes) no matter whether the string literal is normal 
+(enclosed in single quotes) or multi-line (enclosed in triple quotes). 
+Instead, there is are two new forms of dollar sign escape. 
+The most general form encloses an expression in \${ and }, i.e. \${expr}. 
+The expression enclosed in the braces that follow the leading \$ character is of 
+syntactical category BlockExpr. Hence, it can contain multiple statements, 
+and newlines are significant. Single ‘\$’-signs are not permitted in isolation 
+in a interpolated string. A single ‘\$’-sign can still be obtained by doubling the `\$’ 
```

Comment:
replace backtick by fancy quote, since backtick confuses markdown

Reply:
Sorry there are no definition in google. Could you give an example?

Reply:
sorry, I meant ``\$’` should be `‘\$’`